exercises/bob/instructions.md

```diff
@@ -0,0 +1,18 @@
+# Instructions
+
+Your task is to determine what Bob will reply to someone when they say something to him or ask him a question.
+
+Bob only ever answers one of five things:
+
+- **"Sure."**
+  This is his response if you ask him a question, such as "How are you?"
+- **"Whoa, chill out!"**
+  This is his answer if you YELL AT HIM (in all capitals).
+- **"Calm down, I know what I'm doing!"**
+  This is what he says if you yell a question at him.
+- **"Fine. Be that way!"**
+  This is how he responds if you address him without actually saying anything.
+- **"Whatever."**
+  This is what he answers to anything else.
+
+For the purpose of this exercise, you can assume that Bob's conversational partner always follows normal rules regarding sentence punctuation in English.
```

Comment:
This suggests that input is valid grammar and punctuation. But there are test cases where you have to trim beginning and endings. I think that is okay, but the exercises is not explicit about that.

Reply:
Oh, right. Yes. /thinking